tests/test_pytorch/test_dynamic_shapes.py

```diff
@@ -142,6 +142,9 @@ def test_dynamic_shapes_graph_compiler(tmpdir, hpus):
 
 def test_dynamic_shapes_auto_detect_recompilations(tmpdir):
     """Test auto_detect_recompilations tool."""
+    # Close dist pg if initialized.
```

Comment:
do we need this ? Did u see any scenarios where it was open ?

Reply:
Let's address this in next PR...